src/css/relay.css

```
@@ -33,11 +36,51 @@
   font-size: 16px;
 }
 
+.fx-relay-menu-iframe ::-moz-focus-inner,
+.fx-relay-menu-body ::-moz-focus-inner,
 .fx-relay-menu ::-moz-focus-inner,
 .fx-relay-icon ::-moz-focus-inner {
   border: 0;
 }
 
+.fx-relay-menu-body {
+  margin: 0;
+  padding: 0;
+  background: var(--relayGrey10);
+  background-color: var(--relayGrey10);
+  box-sizing: border-box;
+}
+
+.fx-relay-menu-iframe iframe { 
+  border-radius: 4px;
+  background: var(--relayGrey10);
+  background-color: var(--relayGrey10);
+  border: 0;
+  box-shadow: 3px 3px 3px 3px #00000017;
+  animation: fxRelayFadeIn 0.2s ease forwards;
+}
+
+.fx-relay-menu-iframe { 
+  width: 300px;
+  min-height: 60px;
+  top: 47px;
+  right: -15px;
+  position: absolute;
+}
+
+.fx-relay-menu-iframe::before {
+  content: " ";
+  display: block;
+  position: absolute;
+  top: -7px;
+  right: 20px;
+  height: 20px;
+  width: 20px;
+  background: var(--relayGrey10);
+  z-index: 1;
+  transform: rotate(45deg);
```

:superhero: Is it a bird? Is it a plane? No, it's supernit!

```suggestion
  transform: rotate(45deg);
  animation: fxRelayFadeIn 0.2s ease forwards;
```

---

Not a nit. This was bugggggggging me bad. Nice fix. Now everything fades in together. I also added it to `.fx-relay-menu-wrapper`. Added in ed515009a70c4fd3b452eff9955d37539efb41bc